docs/feature_audio.md

```diff
@@ -167,6 +167,32 @@ The available keycodes for audio are:
 
 !> These keycodes turn all of the audio functionality on and off.  Turning it off means that audio feedback, audio clicky, music mode, etc. are disabled, completely.
 
+## Audio Config
+
+| Settings                        | Default              | Description                                                              |
+|---------------------------------|----------------------|--------------------------------------------------------------------------|
+|`AUDIO_PIN`                      | *Not defined*        |Configures the pin that the Solenoid is connected to.                     |
+|`AUDIO_PIN_ALT`                  | *Not defined*        |Configures the default dwell time for the solenoid.                       |
+|`AUDIO_PIN_ALT_AS_NEGATIVE`      | *Not defined*        |Sets the lower limit for the dwell.                                       |
```

Comment:
Looks like these descriptions need to be replaced with some proper ones.

Reply:
whoops